workspaces/ui/src/entrypoints/development.js

```diff
@@ -1,27 +1,33 @@
 import React from 'react';
-import {useParams, useRouteMatch} from 'react-router-dom';
-import {ApiSpecServiceLoader} from '../components/loaders/ApiLoader';
-import {Provider as DebugSessionContextProvider, useMockSession} from '../contexts/MockDataContext';
-import {ApiRoutes} from '../routes';
-import {Provider as BaseUrlContext} from '../contexts/BaseUrlContext';
+import { useParams, useRouteMatch, matchPath } from 'react-router-dom';
+import { ApiSpecServiceLoader } from '../components/loaders/ApiLoader';
+import {
+  Provider as DebugSessionContextProvider,
+  useMockSession,
+} from '../contexts/MockDataContext';
+import { ApiRoutes } from '../routes';
+import { Provider as BaseUrlContext } from '../contexts/BaseUrlContext';
 
 export default function Development(props) {
-
   const match = useRouteMatch();
-  const {sessionId} = useParams();
+  const { sessionId } = useParams();
+  const isCustomerSession = !!matchPath(
```

Comment:
if we compose the components differently, as described in my other comments, then we would not need this here